Modules/Module_Import/NII2BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -117,7 +117,7 @@
 					') & studyPar (' xASL_num2str(jsonOut.(fn{1})) '). '];
 
 				% Define the DICOM or studyPar priority
-				if strcmp(fn{1}, 'TotalAcquiredPairs') || strcmp(fn{1}, 'ArterialSpinLabelingType') || strcmp(fn{1}, 'EchoTime') || strcmp(fn{1}, 'VascularCrushing')
+				if strcmp(fn{1}, 'TotalAcquiredPairs') || strcmp(fn{1}, 'ArterialSpinLabelingType') || strcmp(fn{1}, 'EchoTime') || strcmp(fn{1}, 'VascularCrushing') || strcmp(fn{1}, 'PostLabelingDelay') || strcmp(fn{1}, 'LabelingDuration')
```

Comment:
I don't get this. We made the following rule afaik:

1. studyPar always has priority over DICOM
2. Users should always start with an empty studyPar, check what is missing (i.e. not obtained from the DICOM) and then add this to the studyPar

Unless you can give me a use case where DICOM fields should have priority, and the user cannot simply remove the parameter from the studyPar.

So we should always compare all parameters between DICOM and studyPar.

I'm fine for now if we compare this only for a few and only issue a warning for a few, but we need to always prioritize studyPar, period :)
(for the simple reason that studyPar is only used exceptionally)

Reply:
Yes. That's something old from Michael probably. Because studyPar has priority for all, doesn't need to be checked.

We should make a new issue and check that and fix that in agreement with what you say. 

For now, we just need to accommodate was has to be done for the new dcm2nii version.

Reply:
Agree. New issue created #1865